gatsby-config.js

```diff
@@ -1,11 +1,82 @@
-module.exports = {
-  siteMetadata: {
+module.exports = { 
+  pathPrefix: "/fullstack-hy2020.github.io",
+  siteMetadata: {   
     title: 'Full Stack open 2020',
     description: '',
     author: 'Houston Inc. Consulting oy',
     siteUrl: 'https://fullstack-hy2020.github.io/',
   },
   plugins: [
+    {
```

Comment:
Maybe we could get rid of the copy-paste code by defining and using a function such as `createSearchConfig(name, lang)` this file? At first glance `name` and query's `lang` argument seems to require to be passed as arguments

Reply:
I added create function and tested build. Also added Footer Element to search pages. It seemed that other pages had it also.